app/js/templates/adminTable.hbs

```diff
@@ -1,34 +1,124 @@
 {{#each data}}
 <tr
     {{#if this.topGrade}}
-         class="top-grade"
+         class="top-grade first-row"
     {{else if this.passingGrade}}
-         class="passing-grade"
+         class="passing-grade first-row"
     {{else if this.fail}}
-         class="dubious-grade"
+         class="dubious-grade first-row"
     {{else if this.notTakenYet}}
-         class="test-not-taken"
+         class="test-not-taken first-row"
     {{/if}}>
-        <td dataId="{{id}}" dataName="{{name}}" dataEmail="{{email}}"  dataPercentage="{{percentage}}" dataTime="{{time}}" dataTimeAllowed="{{timeAllowed}}" dataCanRetake="{{canRetake}}" dataTestId="{{this.testId}}">
-            <button class="btn edit-user-button btn_height modalBtn">Edit User</button>
-            <button class="btn delete-user-button btn_height">Delete User</button>
-            {{#unless testNotTaken}}
-               <button class="btn download-user-results-button btn_height">Download Results</button>
-               <button class="btn view-results-button btn_height">View Results</button>
-            {{/unless}}
+    <td dataId="{{id}}" dataName="{{name}}" dataEmail="{{email}}"  dataPercentage="{{percentage}}" dataTime="{{time}}" dataTimeAllowed="{{timeAllowed}}" dataCanRetake="{{canRetake}}" dataTestId="{{this.testId}}">
+       <button class="btn more-info-button btn_height" data-id="{{id}}">More Info</button>
+     </td>
+     <td class="name">{{name}}</td>
+     <td class="email">{{email}}</td>
+     <td class="percentage">{{percentage}}</td>
+     <td class="date"></td>
+</tr>
+<tr data-id="{{id}}"
+        {{#if this.topGrade}}
+            class="top-grade secondRow hide"
+        {{else if this.passingGrade}}
+             class="passing-grade secondRow hide"
+        {{else if this.fail}}
+             class="dubious-grade secondRow hide"
+        {{else if this.notTakenYet}}
+             class="test-not-taken secondRow hide"
+        {{/if}}>
```

Comment:
the class first-row has kebab case whereas secondRow is camel case. make them match, also "first row" and "second row" are not good names for this because it sounds like first-row only refers to the first tow in the table. you can call them something like row-header and row-expanded-info or something like that.

Reply:
done